.jekyll-cache/Jekyll/Cache/Jekyll--Converters--Markdown/c7/71564f874a8bb83404d37c922fd6727a44d1cbcbee928c1d9c82a3c2d52fb6

```diff
@@ -0,0 +1,32 @@
+I"<table>
+  <thead>
+    <tr>
+      <th>Test Instruction</th>
```

Comment:
I think it's clearer to separate the Identify Content from the actual Test Instructions as they are separate sections in the Baseline. Not stuck on whether it needs to be a [separate table like in 5.1-1-fail-1](https://section508coordinators.github.io/baselinealignment/testcases/TC05.1-1-fail-1.html)

Reply:
My preference would be to have one table of test case steps. I think it will just make it easier, and less confusing, to combine once we get to a single alignment validation report.

Reply:
I'm ok with Identify Content being just one row in the Test Instruction table for Baselines that have just one IC and no visible identifier (which is a most I think). For Baseline IC's that have multiple, like Forms 10.1 has 2 numbered ICs, then there can be a row for each. 

I would not permit TC edits of the Baseline instructions. We were very careful about the Baseline wording. Allowing edits may add misinterpretations.

Reply:
Agreed. Will need to resolve that in some of the already published test cases. Since we need to revise to reference the Baseline test instructions anyway, we can fix both issues at the same time.